File: SantanderTeste/app/src/main/java/com/example/alessandrofsouza/santanderapp/MainActivity.java
package com.example.alessandrofsouza.santanderapp;

import android.os.Bundle;
import android.support.design.widget.TabLayout;
import android.support.v4.view.PagerAdapter;
import android.support.v4.view.ViewPager;
import android.support.v7.app.AppCompatActivity;

import com.example.alessandrofsouza.santanderapp.presentation.pages.contact.ContactPresenter;
import com.example.alessandrofsouza.santanderapp.presentation.pages.investment.InvestmentPresenter;
import com.example.alessandrofsouza.santanderapp.presentation.adapters.PageAdapter;

public class MainActivity extends AppCompatActivity {

    private static final String TAG = "Santander ";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        getSupportActionBar().hide();

//        PageAdapter adapter = new PageAdapter(getSupportFragmentManager());
//        adapter.add(new InvestmentPresenter(), getString(R.string.investimento));
//        adapter.add(new ContactPresenter(), getString(R.string.contato));

        PagerAdapter adapter = new PageAdapter(this, getSupportFragmentManager());

        ViewPager viewPager = findViewById(R.id.mainPage);
        viewPager.setAdapter(adapter);

        TabLayout tabLayout = findViewById(R.id.pageTab);
        tabLayout.setupWithViewPager(viewPager);
    }
}

File: app/src/main/java/santander/com/br/invest/ui/MainActivity.kt
package santander.com.br.invest.ui

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*
import santander.com.br.invest.R
import santander.com.br.invest.ui.adapter.ViewPagerAdapter
import santander.com.br.invest.ui.fragments.ContactFragment
import santander.com.br.invest.ui.fragments.InvestmentFragment


class MainActivity : AppCompatActivity() {

  override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)
    setContentView(R.layout.activity_main)

    initViewPager()
  }

  private fun initViewPager() {
    viewPager.offscreenPageLimit = 2
    tabLayout.setupWithViewPager(viewPager)

    val adapter = ViewPagerAdapter(supportFragmentManager)
    adapter.addFragment(InvestmentFragment(), getString(R.string.investment))
    adapter.addFragment(ContactFragment(), getString(R.string.contact))
    viewPager.adapter = adapter
  }
}